src/MigrationStatus.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Class MigrationStatus
+ */
+class MigrationStatus
+{
+    /**
+     * Option key, and option page slug
+     *
+     */
+    private string $key = 'planet4_migration_status';
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        // Set our title.
+        $this->title = 'Migration status';
+        $this->hooks();
+    }
+
+    /**
+     * Register our setting to WP.
+     */
+    public function init(): void
+    {
+        register_setting($this->key, $this->key);
+    }
+
+    /**
+     * Initiate our hooks
+     */
+    public function hooks(): void
+    {
+        add_action('admin_init', [ $this, 'init' ]);
+        add_action('admin_menu', [ $this, 'add_options_page' ], 99);
+    }
+
+    /**
+     * Add menu options page.
+     */
+    public function add_options_page(): void
+    {
+        add_submenu_page(
+            'planet4_settings_navigation',
+            $this->title,
+            $this->title,
+            'manage_options',
+            $this->key,
+            [ $this, 'admin_page_display' ]
+        );
+    }
+
+    /**
+     * Admin page markup. Mostly handled by CMB2.
+     */
+    public function admin_page_display(): void
+    {
+        echo '<h1>P4 Migration Status</h1>' . "\n";
```

Comment:
Let's use `<h2>` here, same as other pages in that section. I think "Development" is the only other section that has `h1`. We should probably change that one too.

Reply:
I think just "Data migrations" is fine